components/ListCripto/index.tsx

```diff
@@ -25,6 +37,7 @@ const ListCripto = ({setModal}: ListCriptoProps) => {
           </PressableAdd>
         }
       />
+      {error !== '' ? errorAlert() : null}
```

Comment:
this should be inside a useEffect, and if the error updates, then show the alert